### DISCUSSION TITLE
Redlining notebook 92 band labels

### DISCUSSION THREAD

**Discussion by hanried** [earthlab-education/Earth-Analytics-AY24#586] [Q&A]
I just finished going through the Part 2 Demo video for our redlining assignment and I am wondering if the band labels got mixed up in notebook 92 in the video.

It is in the code cell where we set up the for loops to process through all the different bands. In the video, we assign B02 to red and B04 to blue:
![image](https://github.com/user-attachments/assets/41502d42-9a7e-4af2-8514-c1a3bf937451)

Back in notebook 32 though, we have B02 assigned to blue and B04 assigned to red:
![image](https://github.com/user-attachments/assets/d0ee5249-5280-4489-b1a8-55f86c1e9f09)

In notebook 92, should B02 be blue and B04 be red?

let me know if those screenshots are too annoying to read!

**Comment by eculler** [comment]
Oh, good catch @hanried ! Looking at the [User Guide](https://lpdaac.usgs.gov/documents/1698/HLS_User_Guide_V2.pdf), B02 is Blue, and B04 is Red, so the original notebook is correct! Maybe that is why the image is so dark...I'll have to check that out.
